spec/lib/forms/choose_your_provider_spec.rb

```diff
@@ -41,28 +41,26 @@
     end
 
     npqeyl_and_npqll_codes = %w[
-      NPQEYL
```

Comment:
Looking back over this makes it clear we had identifiers all along, they were just in the code and stored in COURSE_NAMES. Which wasn't clear.

Reply:
The new system is a far better identifier system since it uses the correct identifier codes everywhere.